source/PyMaterialX/CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ add_subdirectory(PyMaterialXFormat)
 add_subdirectory(PyMaterialXGenShader)
 add_subdirectory(PyMaterialXGenGlsl)
 add_subdirectory(PyMaterialXGenOsl)
+add_subdirectory(PyMaterialXRender)
```

Comment:
This should be added conditionally based on the build flag.
i.e. All Render* modules should be weapped in a if (MATERIALX_BUILD_RENDERGLSL)